### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by yashchn**

### CODE REVIEW

File: experiment/posttest.js [hunk 1]
```diff
@@ -89,25 +89,47 @@
...
     {
-      question: "<img src='images/8.PNG'><br>Identify the location of Secondary electron detector",  ///// Write the question inside double quotes
+      question: "2. Time period of the oscillator is constant for given values of mass, length and initial conditions.",  ///// Write the question inside double quotes
       answers: {
-        a: "<img src='images/1b.png'>",                  ///// Write the option 1 inside double quotes
-        b: "<img src='images/1a.png'>",                  ///// Write the option 2 inside double quotes
-        c: "<img src='images/1c.PNG'>",      },
-      correctAnswer: "c"                ///// Write the correct option inside double quotes
+        a: "True",                  ///// Write the option 1 inside double quotes
+        b: "False'>",                  ///// Write the option 2 inside double quotes
```
Comment (raj-vlabs) [inline experiment/posttest.js]: Why is this right chevron ('>) required in this and a few other answers after it? Did you actually test the experiment after coding?

File: experiment/references.md [hunk 2]
```diff
@@ -1 +1,4 @@
-### Link your references in here
\ No newline at end of file
+- [http://en.wikipedia.org/wiki/Simple_harmonic_motion](http://en.wikipedia.org/wiki/Simple_harmonic_motion)
```
Comment (raj-vlabs) [inline experiment/references.md:1]: Please use https links as much as possible

File: experiment/simulation/index.html [hunk 3]
```diff
@@ -1,13 +1,80 @@
...
+					<div class="column has-text-centered">
+						<div id ="plot2"></div>
+					</div>
+				</div>
+			</div>
+		</div>
+
+		<!-- Add JS at the bottom of HTML file --> 
+		<script src="https://cdn.jsdelivr.net/npm/animejs@3.0.1/lib/anime.min.js"></script>
+		<script src="path/to/anime.min.js"></script>
```
Comment (raj-vlabs) [inline experiment/simulation/index.html]: Why is this line still there?

File: experiment/simulation/css/main.css [hunk 4]
```diff
@@ -1 +1,72 @@
-/* You CSS goes in here */
\ No newline at end of file
+.page {
+	overflow-x: scroll;
+	overflow-y: scroll;
+}
+
+.but {
```
Comment (raj-vlabs) [inline experiment/simulation/css/main.css]: Please rename this class to use a more meaningful name. but doesn't really tell me anything

File: experiment/simulation/js/main.js [hunk 5]
```diff
@@ -1 +1,223 @@
...
+	restartButton.addEventListener('click', function() { myAnimation.restart(); });
+
+	function updatePara()
+	{
+		displacement = document.getElementById("displacement").value;
+		height = document.getElementById("height").value;
+		mass = document.getElementById("mass").value;
+
+		const myElement = document.querySelector("#rod");
+		const myElement1 = document.querySelector("#ball");
```
Comment (raj-vlabs) [inline experiment/simulation/js/main.js]: Please rename both of these variables to have meaningful names.

File: experiment/simulation/js/main.js [hunk 6]
```diff
@@ -1 +1,223 @@
...
+			const layout = {
+				// title: {
+				//   text:'Plot Title',
+				//   font: {
+				//     family: 'Courier New, monospace',
+				//     size: 24
+				//   },
+				//   xref: 'paper',
+				//   x: 0.05,
+				// },
```
Comment (raj-vlabs) [inline experiment/simulation/js/main.js]: Why is the dead code not removed?

**Review by raj-vlabs — CHANGES_REQUESTED** [review]
Please take a look at the comments I posted on the slack channel. I do not expect to individually address people to tell them the same things.

File: experiment/simulation/index.html (hunk 3 above)
Comment (SRDewan) [reply experiment/simulation/index.html]: We are using animejs in this experiment.

Comment (raj-vlabs) [reply experiment/simulation/index.html]: And it is being loaded from 2 locations, one of which is 'path/to/anime.min.js' ??

**Pull request closed without merging**